tests/test_bot.py

```diff
@@ -2964,3 +2965,18 @@ def test_coroutine_functions(self):
             "The following methods should be defined as coroutine functions: "
             f"{','.join(non_coroutine_functions)} "
         )
+
+    def test_api_kwargs_present(self):
```

Comment:
why not additionally check for the timeout kwargs? Or are we already testing that somewhere

Reply:
Don't recall that :D but I'll extend the tests 👍